### ISSUE TITLE
`ActiveEffectConfig` fails to show default image or localize alt text

### ISSUE THREAD

**Issue by stwlam** [foundryvtt/foundryvtt#11933]
Originally reported by [asheeon](https://discord.com/channels/170995199584108546/1296292044011995136/1311076872125288510)

**Short Description of bug**: ActiveEffect does not set default img & ActiveEffect Config's img alt text is not getting localized.
**Simple steps to reproduce the bug**: 
```js
const actor = await Actor.create({name: 'test actor', type: 'base'});
const ae = await ActiveEffect.create({name: 'test effect', type: 'base'}, {parent: actor});
ae.sheet.render(true)
```
**Screenshots and/or console errors**:
![no-img](https://github.com/user-attachments/assets/24cc91ae-e20b-4e5b-976b-104f08a96d1a)
